arcgis-ios-sdk-samples/Maps/Show location history/README.metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "category": "Maps",
+    "description": "This sample shows you how to display your location history on the map.",
+    "ignore": false,
+    "images": [
+        "image1.png"
+    ],
+    "keywords": [
```

Comment:
You'll want to add the Tags from the README to this array of strings.

Reply:
I have tidied this up via [`e8d8235`](https://github.com/Esri/arcgis-runtime-samples-ios/pull/751/commits/e8d82357ea8675704803bc542acafc6859f8d3e9).